.github/workflows/kedro-docs-only.yml

```diff
@@ -0,0 +1,24 @@
+name: Run checks on Kedro Docs
```

Comment:
Maybe redundant with Read the Docs?

Reply:
Following the current behaviour for now. I think read the docs errors are very hard to find 😅